lib/db/StudyDAO.js

```diff
@@ -53,7 +53,7 @@ SELECT
     const sql = `
 INSERT INTO "studies" (
   "createdAt",
-  "userSubject",
+  "userId",
```

Comment:
need to test these changes end-to-end.  can we add REST API tests for associated endpoints at some point?

Reply:
punting REST API tests here to future iterations